### PR TITLE
refactor: allow no I and jsx lambdas

### PR DESCRIPTION
**Pull request by weblancaster**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -106,12 +106,17 @@ module.exports = {
     "member-access": [
       false
     ],
+    "interface-name": [
```
Comment (weblancaster) [inline index.js:109]: we have only two options `always allow` or `never allow`

File: index.js [hunk 2]
```diff
@@ -106,12 +106,17 @@ module.exports = {
...
+    ],
     // Prettier
     "prettier": true,
     // TS React plugin
     "jsx-wrap-multiline": true,
     "jsx-curly-spacing": "never",
     "jsx-boolean-value": "never",
-    "jsx-equals-spacing": "never"
+    "jsx-equals-spacing": "never",
+    "jsx-no-lambda": true
```
Comment (weblancaster) [inline index.js:120]: the advantages of not using lambdas in jsx are too small at the moment.

**Review by juliangieseke — APPROVED** [review]

**Review by Poltergeist — APPROVED** [review]

**Review by DanielMSchmidt — CHANGES_REQUESTED** [review]
Could you please pull the JIRA for this into the sprint and link it here?

**Comment by weblancaster** [comment]
@DanielMSchmidt I did not find any Jira, is there one? if not let me know so I can create. Thank you.

**Comment by weblancaster** [comment]
@DanielMSchmidt can we dismiss your request changes? or if you know there is a Jira please send to me so I can update the PR

**Review by DanielMSchmidt — APPROVED** [review]

**Comment by DanielMSchmidt** [comment]
@weblancaster Thank you for giving me a reminder, totally forgot that I wanted to unblock this 🙈

**Pull request merged**